### PR TITLE
Rename Repbase direcotry to avoid confusing caused

### PR DESCRIPTION
**Pull request by luzengAdelaide**
@dadelson @kortschak Revised Joy's code in GenerateAnnotatedLibrary.java, the original code point to RepBase libraries caused confusing.

### CODE REVIEW

File: code/GenerateAnnotatedLibrary.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 		SSR.txt (list of sequences identified as SSRs, and the SSR they matched)
 		protein.txt (list of the sequences identified as proteins and the protein they matched)
 		//LA4v2-satellite.fa (a satellite sequence for which no consensus sequence was found)
-		/home/a1635743/RepBase20.04.fasta/*rep.ref (RepBase libraries to base 
+		/home/a1635743/RepBase_lib/*.rep.ref (RepBase libraries to base
```
Comment (kortschak) [inline code/GenerateAnnotatedLibrary.java:23]: What's the basis for this change?

File: code/GenerateAnnotatedLibrary.java~ [hunk 2]
```diff
@@ -0,0 +1,674 @@
+import java.io.*;
```
Comment (kortschak) [inline code/GenerateAnnotatedLibrary.java~:1]: Why is this file added?

File: code/GenerateAnnotatedLibrary.java (hunk 1 above)
Comment (luzengAdelaide) [reply code/GenerateAnnotatedLibrary.java:23]: @kortschak James thought this is a fasta file instead of a directory that contains RepBase library.

**Comment by luzengAdelaide** [comment]
This was unnecessary!
